products/dashboard/.env.example

```
@@ -15,3 +15,10 @@ SENTRY_RELEASE=local-dev
 SENTRY_AUTH_TOKEN=sntrys_<token>
 SENTRY_ORG=<org-slug>
 SENTRY_PROJECT=<project-slug>
+
+# PostHog — EU region
+# Project token is public (NEXT_PUBLIC prefix is intentional and correct)
+# Host must always be the EU endpoint — do not change to us.i.posthog.com
+# phc_... from PostHog project settings
+NEXT_PUBLIC_POSTHOG_PROJECT_TOKEN=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)